CONTRIBUTING.md

```diff
@@ -2,26 +2,25 @@
   Thx for your interest! We're so glad you're here. 
 
 ### Important Resources
-  - docs: [https://github.com/packethost/standards/wiki](https://github.com/packethost/standards/wiki)
-  - bugs: [https://github.com/packethost/standards/issues](https://github.com/packethost/standards/issues)
+  - bugs: [https://github.com/equinix-labs/equinix-labs/issues](https://github.com/equinix-labs/equinix-labs/issues)
 
 ### Code of Conduct
-Available via [https://github.com/packethost/standards/blob/main/CODE_OF_CONDUCT.md](https://github.com/packethost/standards/blob/main/CODE_OF_CONDUCT.md)
+Available via [https://github.com/equinix-labs/equinix-labs/blob/main/CODE_OF_CONDUCT.md](https://github.com/equinix-labs/equinix-labs/blob/main/CODE_OF_CONDUCT.md)
 
 ### Testing
 Wait, we're supposed to set up testing?
```

Comment:
I don't know what this means and it sounds weird to me?

Reply:
I could see removing this CONTRIBUTING.md entirely. I do wonder what precedent we are setting though, by saying all projects in equinix-labs _must_ contain certain files, when the repository that sets that standard does not abide.

Reply:
This file is performing a strange role of offering guidance for maintenance of the 'standards' project while describing aids for other projects that are contained in the 'standards' project.

I'll take a pass at restructuring this file.